### ISSUE TITLE
Kexec upgrade fails on v0.8.1

### ISSUE THREAD

**Issue by janeczku** [rancher/os#1712]
**RancherOS Version: (ros os version)**
v0.8.1

Steps:

1. Run `sudo ros os upgrade --kexec`

Result:

1.  Error in the command output
2. After reboot host does not come up again correctly (console never starts)

```
$ sudo ros os upgrade --kexec
Upgrading to rancher/os:v0.9.0
Continue [y/N]: y
Pulling os-upgrade (rancher/os:v0.9.0)...
v0.9.0: Pulling from rancher/os

627beaf3eaaf: Pull complete 
c62794cdced6: Pull complete 
c4626eff5f27: Pull complete 
b47e38b4585d: Pull complete 
4914c31b1e0c: Pull complete 
172683e78e1a: Pull complete 
865dbcf06d52: Pull complete 
df311c14e097: Pull complete 
41861a405f90: Pull complete 
Digest: sha256:f48961131d180cebfc55a324e89316e35227ca43025cb74886d5929b76cb21ee
Status: Downloaded newer image for rancher/os:v0.9.0
os-upgrade_1 | Incorrect Usage.
os-upgrade_1 | 
os-upgrade_1 | NAME:
os-upgrade_1 |    ros install - install RancherOS to disk
os-upgrade_1 | 
os-upgrade_1 | USAGE:
os-upgrade_1 |    ros install [command options] [arguments...]
os-upgrade_1 | 
os-upgrade_1 | OPTIONS:
os-upgrade_1 |    --image value, -i value         install from a certain image (e.g., 'rancher/os:v0.7.0')
os-upgrade_1 |                                                use 'ros os list' to see what versions are available.
os-upgrade_1 |    --install-type value, -t value  generic:    (Default) Creates 1 ext4 partition and installs RancherOS (syslinux)
os-upgrade_1 |                         amazon-ebs: Installs RancherOS and sets up PV-GRUB
os-upgrade_1 |                         gptsyslinux: partition and format disk (gpt), then install RancherOS and setup Syslinux
os-upgrade_1 |    --cloud-config value, -c value  cloud-config yml file - needed for SSH authorized keys
os-upgrade_1 |    --device value, -d value        storage device
os-upgrade_1 |    --force, -f                     [ DANGEROUS! Data loss can happen ] partition/format without prompting
os-upgrade_1 |    --no-reboot                     do not reboot after install
os-upgrade_1 |    --append value, -a value        append additional kernel parameters
os-upgrade_1 |    --kexec                         reboot using kexec
os-upgrade_1 |    --debug                         Run installer with debug output
os-upgrade_1 |    
Continue with reboot [y/N]: y
> INFO[0025] Rebooting

```

**Comment by SvenDowideit** [comment]
this is going to require a kexec package for alpine - so probably will be post 0.9.2

**Comment by SvenDowideit** [comment]
current progress - building an alpine kexec doesn't work, it might need a kexec build for our kernels - trying debian and building

**Comment by tfiduccia** [comment]
I ran the ros os upgrade --kexec in v0.9.2-rc1  and although I can re-log into the rancheros machine, it's still giving me that error and it's not going to v0.8.2
```
[rancher@ip-172-31-25-36 ~]$ sudo ros os upgrade --kexec -i rancher/os:v0.8.2
Upgrading to rancher/os:v0.8.2
Continue [y/N]: y
Pulling os-upgrade (rancher/os:v0.8.2)...
v0.8.2: Pulling from rancher/os
627beaf3eaaf: Already exists 
fa9e34f657f9: Pull complete 
b4cb57fc58d8: Pull complete 
a72ddf4cf24b: Pull complete 
e25ab8f0c651: Pull complete 
e1a90864d117: Pull complete 
78fcb912d8e3: Pull complete 
73b5f5d5261e: Pull complete 
3dfd17477527: Pull complete 
Digest: sha256:51e6f3f5300eac9a5576bb72935c6e9c59a2cc879fc0cfc00fb87b9efea10dc0
Status: Downloaded newer image for rancher/os:v0.8.2
os-upgrade_1 | Incorrect Usage.
os-upgrade_1 | 
os-upgrade_1 | NAME:
os-upgrade_1 |    ros install - install RancherOS to disk
os-upgrade_1 | 
os-upgrade_1 | USAGE:
os-upgrade_1 |    ros install [command options] [arguments...]
os-upgrade_1 | 
os-upgrade_1 | OPTIONS:
os-upgrade_1 |    --image value, -i value         install from a certain image (e.g., 'rancher/os:v0.7.0')
os-upgrade_1 |                                                use 'ros os list' to see what versions are available.
os-upgrade_1 |    --install-type value, -t value  generic:    (Default) Creates 1 ext4 partition and installs RancherOS (syslinux)
os-upgrade_1 |                         amazon-ebs: Installs RancherOS and sets up PV-GRUB
os-upgrade_1 |                         gptsyslinux: partition and format disk (gpt), then install RancherOS and setup Syslinux
os-upgrade_1 |    --cloud-config value, -c value  cloud-config yml file - needed for SSH authorized keys
os-upgrade_1 |    --device value, -d value        storage device
os-upgrade_1 |    --force, -f                     [ DANGEROUS! Data loss can happen ] partition/format without prompting
os-upgrade_1 |    --no-reboot                     do not reboot after install
os-upgrade_1 |    --append value, -a value        append additional kernel parameters
os-upgrade_1 |    --kexec                         reboot using kexec
os-upgrade_1 |    --debug                         Run installer with debug output
os-upgrade_1 |    
Continue with reboot [y/N]: y
> INFO[0057] Rebooting                                    
```

**Comment by SvenDowideit** [comment]
resolved in 0.9.2-rc4
